modules/core/server/views/layout.server.view.html

```diff
@@ -45,7 +45,17 @@
 
   <!--Embedding The User Object-->
   <script type="text/javascript">
-    var user = {{ user | json | safe }};
+    var user = {
+      displayName: "{{ user.displayName | escape }}",
```

Comment:
the user object comes from the server's response object/express variables.
that view is server-side view, not angular.

on your comment with not setting it, how about setting to an empty object rather than null/undefined? is that safer?